services/app-web/src/pages/StateSubmission/SubmissionType/SubmissionType.tsx

```diff
@@ -193,27 +203,137 @@ export const SubmissionType = ({
+                updatedDraftSubmissionFormData.modifiedNetworkAdequacyStandards =
+                    draftSubmission.draftRevision.formData.modifiedNetworkAdequacyStandards
+                updatedDraftSubmissionFormData.modifiedLengthOfContract =
+                    draftSubmission.draftRevision.formData.modifiedLengthOfContract
+                updatedDraftSubmissionFormData.modifiedNonRiskPaymentArrangements =
+                    draftSubmission.draftRevision.formData.modifiedNonRiskPaymentArrangements
+            } else {
+                updatedDraftSubmissionFormData.inLieuServicesAndSettings =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedBenefitsProvided =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedGeoAreaServed = undefined
+                updatedDraftSubmissionFormData.modifiedMedicaidBeneficiaries =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedRiskSharingStrategy =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedIncentiveArrangements =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedWitholdAgreements =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedStateDirectedPayments =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedPassThroughPayments =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedPaymentsForMentalDiseaseInstitutions =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedMedicalLossRatioStandards =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedOtherFinancialPaymentIncentive =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedEnrollmentProcess =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedGrevienceAndAppeal =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedNetworkAdequacyStandards =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedLengthOfContract =
+                    undefined
+                updatedDraftSubmissionFormData.modifiedNonRiskPaymentArrangements =
+                    undefined
+            }
 
             try {
-                const updatedDraft = await updateDraft(draftSubmission)
+                const updatedContractInput: UpdateContractDraftRevisionInput = {
+                    formData: updatedDraftSubmissionFormData,
+                    contractID: draftSubmission.id,
+                    lastSeenUpdatedAt: draftSubmission.draftRevision.updatedAt,
+                }
+                const updatedDraft = await updateDraft(updatedContractInput)
                 if (updatedDraft instanceof Error) {
```

Comment:
can this throw anymore? I thought we wrapped that in the updateDraft call so we could skip try/catch here?